configTools/src/test/scala/com/gu/janus/testutils/RightValues.scala

```diff
@@ -0,0 +1,24 @@
+package com.gu.janus.testutils
+
+import org.scalactic.source.Position
+import org.scalatest.exceptions.{StackDepthException, TestFailedException}
+
+trait RightValues {
```

Comment:
This restore `EitherValues` behaviour that works with fully Right-biased Eithers. Scala 2.13 deprecates using the Right projection, which is what EitherValues still requires us to use. `RightValues` adds `.value` to Either types directly.

Old way
```scala
val foo: Either[L, R] = ???
foo.right.value shouldEqual expected
```

New way
```scala
val foo: Either[L, R] = ???
foo.value shouldEqual expected
```

Handling Left values can still be done using `EitherValues`, because the LeftProjection still works fine.